### PR TITLE
fix: [#1079] adding max width to menu items and also the tooltip to show full text value

### PR DESCRIPTION
**Pull request by Daggerhub**
Screenshot 2023-08-05 at 12 40 58 PM
Screenshot 2023-08-05 at 12 41 09 PM

### CODE REVIEW

**Comment by charlesBochet** [comment]
@Daggerhub we actually wanted to fix the dropdown width. I've done it in this PR: https://github.com/twentyhq/twenty/pull/1093

I believe you can rebase on main and work from there with  OverflowingTextWithTooltip as you are already doing, should be straighforward!

**Comment by Daggerhub** [comment]
Hey @charlesBochet,
Have rebased my branch against main,
Please have a look.

File: front/src/modules/ui/dropdown/components/DropdownMenu.tsx [hunk 1]
```diff
@@ -16,6 +16,8 @@ export const DropdownMenu = styled.div<{
 
   flex-direction: column;
 
+  max-width: 160px;
```
Comment (charlesBochet) [inline front/src/modules/ui/dropdown/components/DropdownMenu.tsx:19]: this is not needed anymore as we are providing width on line 23

**Review by charlesBochet — APPROVED** [review]
Approved with minor comment, thanks!

File: front/src/modules/ui/dropdown/components/DropdownMenu.tsx (hunk 1 above)
Comment (Daggerhub) [reply front/src/modules/ui/dropdown/components/DropdownMenu.tsx]: Have removed it.

**Comment by charlesBochet** [comment]
@Daggerhub your tests are failing. I believe that you need to update the css selector in People.sortBy.stories
To run tests locally:
cd front && yarn storybook-dev # from here, you can go to localhost:6006 and see interactions tests, they should be failing on People.sortBy

in another terminal:
cd front && yarn coverage

**Review by charlesBochet — APPROVED** [review]

**Comment by charlesBochet** [comment]
You seem to have the same issue with Companies.sortBy!

**Review by emilienchvt — APPROVED** [review]

**Comment by charlesBochet** [comment]
Thank you! Merging!

**Pull request merged**